src/main/resources/com/microsoftopentechnologies/windowsazurestorage/AzureBlobProperties/config.jelly

```diff
@@ -14,4 +14,11 @@
     <f:entry title="${%contentType_title}" field="contentType">
         <f:textbox />
     </f:entry>
+
+    <f:entry field="detectContentType">
+        <div align="left">
+            <f:checkbox title="${%detectContentType_title}" />
```

Comment:
OK, updated. But for existing jobs it will remain false as the config has already been persisted.